TwitchDownloaderCore/Chat/ChatHtml.cs

```diff
@@ -69,11 +69,11 @@ public static async Task SerializeAsync(string filePath, ChatRoot chatRoot, bool
                         }
                         break;
                     case "<!-- CUSTOM HTML -->":
-                        foreach (Comment comment in chatRoot.comments)
+                        foreach (var comment in chatRoot.comments)
                         {
                             var relativeTime = new TimeSpan(0, 0, (int)comment.content_offset_seconds);
                             string timestamp = relativeTime.ToString(@"h\:mm\:ss");
-                            await sw.WriteAsync($"<pre class=\"comment-root\">[{timestamp}] {GetChatBadgesHtml(embedData, chatBadgeData, chatRoot, comment)} <a href=\"https://www.twitch.tv/{comment.commenter.name}\" target=\"_blank\"><span class=\"comment-author\" {(comment.message.user_color == null ? "" : $"style=\"color: {comment.message.user_color}\"")}>{(comment.commenter.display_name.Any(x => x > 127) ? $"{comment.commenter.display_name} ({comment.commenter.name})" : comment.commenter.display_name)}</span></a><span class=\"comment-message\">: {GetMessageHtml(embedData, thirdEmoteData, chatRoot, comment)}</span></pre>\n");
```

Comment:
Did we not want to open links in an external tab anymore? (the removal of target="_blank"

Reply:
I moved it to the head of `template.html` so we don't need to define it for every comment on the page. Saves a fair amount of file size.

https://github.com/lay295/TwitchDownloader/blob/620d000fe69a5580ca4507228ee95200ad115ff1/TwitchDownloaderCore/Resources/template.html#L2-L6

Reply:
Oops didn't see that, cool.